pages/osx/pwgen.md

```diff
@@ -0,0 +1,15 @@
+# pwgen
+
+> Generate pronounceable passwords.
+
+- Generate random password with s[y]mbols:
```

Comment:
What's `y` doing in square brackets here?

Reply:
Unsure, it came from the original from linux, is it best to make a new PR to remove it from there, or do it from both here?

Reply:
Haha, its a convention we came up with. If the parameter in the example is a single letter, then at least a word in the description has to start with it.

Here, the parameter is `-y` and it stands for symbol. Which is weird. Hence [y] to denote the `-y`.

Reply:
Ahh cool! Haven't seen that elsewhere.